src/wp-includes/class-wp-object-cache.php

```diff
@@ -499,12 +499,18 @@ public function flush() {
 	 * Sets the list of global cache groups.
 	 *
 	 * @since 3.0.0
+   * @since 6.0.0 Added the possibility to ignore global groups by setting the WP_OBJECT_CACHE_IGNORE_GLOBAL_GROUPS as an array in wp-config.php   
 	 *
 	 * @param string|string[] $groups List of groups that are global.
 	 */
 	public function add_global_groups( $groups ) {
 		$groups = (array) $groups;
 
+		// Allow force ignoring of global groups.
+		if ( defined( 'WP_OBJECT_CACHE_IGNORE_GLOBAL_GROUPS' ) && ! empty( WP_OBJECT_CACHE_IGNORE_GLOBAL_GROUPS ) ) {
+			$groups = array_diff( $groups, (array) WP_OBJECT_CACHE_IGNORE_GLOBAL_GROUPS );
```

Comment:
I would advise against type casting the constant to an array. Instead, there are 2 ways to go:

1. Check that the constant is an `array` data type and skip the `array_diff()` if it's not.
2. Assume the developer used the proper data type and let PHP handle it natively by throwing an exception if it's not.

Which option? I'd suggest option 2 as it'll provide feedback to help developers identify their code is doing it wrong.

Reply:
Also, is `defined()` needed here as the constant will be declared by default in `wp-includes/default-constants.php`?